packages/web/src/features/manage-club/funding/component/NewFundingListSection.tsx

```diff
@@ -0,0 +1,109 @@
+"use client";
+
+import React, { useState } from "react";
+import styled from "styled-components";
+
+import Button from "@sparcs-clubs/web/common/components/Button";
+import colors from "@sparcs-clubs/web/styles/themes/colors";
+import Icon from "@sparcs-clubs/web/common/components/Icon";
+import Info from "@sparcs-clubs/web/common/components/Info";
+
+import {
+  newFundingListSectionInfoText,
+  newFundingListSectionTitle,
+  newFundingOrderButtonText,
+} from "@sparcs-clubs/web/constants/manageClubFunding";
+
+import FoldableSectionTitle from "./_atomic/FoldableSectionTitle";
+import NewFundingListTable from "./_atomic/NewFundingListTable";
+
+const NewFundingListSectionInner = styled.div`
+  /* Auto layout */
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 20px;
+
+  width: 100%;
+
+  /* Inside auto layout */
+  /* flex: none;
+  order: 1;
+  flex-grow: 0; */
+`;
+
+const NewFundingListSectionContents = styled.div`
+  /* Auto layout */
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding-left: 24px;
+  gap: 20px;
+
+  width: 100%;
+
+  /* Inside auto layout */
+  /* flex: none;
+  order: 1;
+  align-self: stretch;
+  flex-grow: 0; */
+`;
+
+const NewFundingOrderButtonRow = styled.div`
+  /* Auto layout */
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: center;
+
+  width: 100%;
+
+  /* Inside auto layout */
+  /* flex: none;
+  order: 1;
+  align-self: stretch;
+  flex-grow: 0; */
+`;
+
+const NewFundingOrderButtonText = styled.div`
+  font-family: ${({ theme }) => theme.fonts.FAMILY.PRETENDARD};
+  font-weight: ${({ theme }) => theme.fonts.WEIGHT.MEDIUM};
+  color: ${({ theme }) => theme.colors.WHITE};
+
+  padding-left: 4px;
+`;
+
+const NewFundingListSection: React.FC = () => {
+  const [toggle, setToggle] = useState<boolean>(true);
+  return (
+    <NewFundingListSectionInner>
+      <FoldableSectionTitle
+        title={newFundingListSectionTitle}
+        toggle={toggle}
+        toggleHandler={() => {
+          setToggle(!toggle);
+        }}
+      />
+      {toggle ? (
+        <NewFundingListSectionContents>
+          {/* TODO: 지원금 신청 기간 받아오는 API 생기면 대체해야함! */}
+          <Info text={newFundingListSectionInfoText("2024 봄", new Date())} />
+          <NewFundingOrderButtonRow>
+            <Button>
+              <Icon type="add" size={16} color={colors.WHITE} />
+              <NewFundingOrderButtonText>
+                {newFundingOrderButtonText}
+              </NewFundingOrderButtonText>
+            </Button>
+          </NewFundingOrderButtonRow>
+          {/* TODO: ManageClubFundingMainFrame으로부터 주입받은 테이블 데이터 전달하기 */}
+          <NewFundingListTable />
+        </NewFundingListSectionContents>
+      ) : (
+        <div />
+      )}
```

Comment:
💊 toggle로 삼항연산자 쓰는 대신 and 써서 이렇게 하면 어떨까요? <div/>가 필요한 상황이 아니니 이렇게 하면 불필요하게 <div/> 렌더링 할 필요도 없고 코드도 약간 더 간결해질 것 같아요
```suggestion
      {toggle && (
        <NewFundingListSectionContents>
          {/* TODO: 지원금 신청 기간 받아오는 API 생기면 대체해야함! */}
          <Info text={newFundingListSectionInfoText("2024 봄", new Date())} />
          <NewFundingOrderButtonRow>
            <Button>
              <Icon type="add" size={16} color={colors.WHITE} />
              <NewFundingOrderButtonText>
                {newFundingOrderButtonText}
              </NewFundingOrderButtonText>
            </Button>
          </NewFundingOrderButtonRow>
          {/* TODO: ManageClubFundingMainFrame으로부터 주입받은 테이블 데이터 전달하기 */}
          <NewFundingListTable />
        </NewFundingListSectionContents>
      ) }
```

Reply:
조건부 렌더링에서 두가지 표현 모두 렌더링에 차이가 없는 것으로 알아요!
다만 아래가 좀더 JS스러운? 활용이라고 생각이 되는데, AND 연산 에서 인수가 boolean이 아닌점에 거부감이 들어서 삼항연산자를 쓰고있습니다...ㅎ

Reply:
그런 이유가 있었군요 렌더링은 false일 때 굳이 div가 렌더링 될 필요가 있나 해서 말씀드렸습니다!